pkg/btree/multi.go

```diff
@@ -97,6 +99,60 @@ func (m *LinkedMetaPage) MarshalMetadata(bm encoding.BinaryMarshaler) error {
 	return m.SetMetadata(buf)
 }
 
+func (m *LinkedMetaPage) NextNOffsets() ([]uint64, error) {
```

Comment:
Hmm I see what you're getting at here but I wouldn't expect this to actually traverse the linked list.

Let's pass the offsets into `SetNextNOffsets(offsets []uint64)` instead, and in the caller manage the list of offsets. In other words, make the `NextNOffsets` memory block "dumb" in that it only stores a list of up to 16 items. That way this API stays consistent with the other setters and getters.